apps/rotm/behaviours/save-image.js

```diff
@@ -30,7 +29,7 @@ module.exports = superclass => class extends superclass {
         })
         .catch((err) => {
           req.log('debug', 'Image not saved to S3');
-          next(new ValidationError('unsaved', err));
```

Comment:
This error could be returned by the virus scanner. The user may or may not have intentionally uploaded the virus. In the former case we might not want the user to know that we have detected the virus because by informing them we give them some detail about our security. (i.e. our virus scanner can detect viruses of that nature). In the latter case a user we do not want to discourage the user from using the service, so some info regarding the particular reason they are unable to upload the file could be of value to both parties.

In either case we must provide a balanced response, of which a validation error seemed most suitable. 
Disclaimer:  I don't know how many users might inadvertently upload a virus. Probably not many.

Reply:
It *might* be a virus, in which case we *might* want to catch that particular response and handle it, but as it stands what we display is a message saying:

> Please fix the following error
> We have not been able to upload the photo or screenshot

Obviously this is of no help to the user at all.

Reply:
I agree. It's no help

Reply:
Do you agree that it's worth getting an outside opinion and that a custom message would be more valuable than an application error?

Reply:
We could, but I don't see it as in scope for this PR or the parent ticket. At present the error is mis-handled, so throws a secondary error which loses any metadata from the original error, but also still manifests as a generic application error.

FWIW, when doing this in firearms, any anti-virus error is manifested as a generic application error.